<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QWidget" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>712</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="HeadLabel" name="icon">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="name">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="level">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(196, 95, 125);</string>
         </property>
         <property name="text">
          <string>level:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="title">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(196, 95, 125);</string>
         </property>
         <property name="text">
          <string>title</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="exp">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>exp</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="signButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
         </property>
         <property name="text">
          <string>打哔卡</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="toolButton11">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
         </property>
         <property name="text">
          <string>我的评论</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton4">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
         </property>
         <property name="text">
          <string>我的收藏</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton5">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
         </property>
         <property name="text">
          <string>历史记录</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton0">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
           </property>
           <property name="text">
            <string>主页</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton1">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
           </property>
           <property name="text">
            <string>搜索</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
           </property>
           <property name="text">
            <string>分类</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton3">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
           </property>
           <property name="text">
            <string>排行</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton6">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
           </property>
           <property name="text">
            <string>下载</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton7">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
           </property>
           <property name="text">
            <string>留言板</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton8">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
           </property>
           <property name="text">
            <string>聊天室</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton9">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
           </property>
           <property name="text">
            <string>锅贴</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton10">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 正常时，按钮颜色 */
QToolButton{
    background:transparent;
    color: rgb(196, 95, 125);
    border-style: outset;
    border-color: beige;
    border-radius: 10px;
}

/* 鼠标在按钮上时，按钮颜色 */
 QToolButton:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 10px;
    color: rgb(0, 0, 0);
}

 /* 按钮被单击后，按钮颜色 */
QToolButton:checked {
    background-color:rgb(42, 115, 175);
    border-radius: 10px;
    color: rgb(255, 255, 0);
}</string>
           </property>
           <property name="text">
            <string>游戏区</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">#stackedWidget {border:1px solid #014F84;}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeadLabel</class>
   <extends>QLabel</extends>
   <header location="global">.head_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>signButton</sender>
   <signal>clicked()</signal>
   <receiver>User</receiver>
   <slot>Sign()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>-12</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Switch()</slot>
  <slot>Sign()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
